analytics_data_api/v0/tests/views/test_courses.py

```diff
@@ -3,29 +3,41 @@
 # change for versions greater than 1.0.0. Tests target a specific version of the API, additional tests should be added
 # for subsequent versions if there are breaking changes introduced in those versions.
 
+from __future__ import absolute_import
+
 import datetime
-from itertools import groupby
-import urllib
 from collections import OrderedDict
+from itertools import groupby
 
-import ddt
+import pytz
+import six
+import six.moves.urllib.error  # pylint: disable=import-error
+import six.moves.urllib.parse  # pylint: disable=import-error
+import six.moves.urllib.request  # pylint: disable=import-error
```

Comment:
I think `parse` is the only one of these three actually used in the file, `six` was just playing it conservative and adding imports for everything in urllib that could have been used.